electrum/gui/qt/util.py

```diff
@@ -653,7 +653,11 @@ def add_copy_menu(self, menu, idx):
         for column in self.Columns:
             column_title = self.model().horizontalHeaderItem(column).text()
             item_col = self.model().itemFromIndex(idx.sibling(idx.row(), column))
-            column_data = item_col.text().strip()
+            data = item_col.data()
```

Comment:
The name `data` is not very descriptive, I suggest renaming to `clipboard_data`.
I would rather not use `QStandardItem.setData()` and `QStandardItem.data()` without specifying the `role` argument.
At the top of this class, `MyTreeView`, let's define an attribute (of the class), e.g.:
`ROLE_CLIPBOARD_DATA = Qt.UserRole + 100`
Then, we could have:
```suggestion
            clipboard_data = item_col.data(self.ROLE_CLIPBOARD_DATA)
```

Reply:
Makes sense, done.